apps/jobs/urls.py

```diff
@@ -110,6 +110,11 @@
         views.update_submission_started_at,
         name="update_submission_started_at",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/submissions/(?P<submission_pk>[0-9]+)/cancel/$",
+        views.change_submission_status_to_cancel,
```

Comment:
We should create a generic API that changes the status of the submissions to other status as well.

Reply:
This API is specific to the user functionality of cancelling submission. Why should this one be allowed to set other statuses?